android/build.gradle

```diff
@@ -1,39 +1,40 @@
 def safeExtGet(prop, fallback) {
-  rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
```

Comment:
Can you reverse all the whitespace changes (and update your editor), to make it easier when we want to merge the latest version of RNSound into our fork

Reply:
The space changes are not due to my editor, I made a copy of the version used in react-native-tts (which uses proper spacing (4) for gradle files).

I would prefer my version due to the following (but if you still did agree I will change them):
The section in lines 1–14 does not exist in the original and was added in the previous change with wrong spacing [PR](https://github.com/tomtom-international/react-native-sound/pull/7).
The spaces in lines 19 and 20 before that PR are in the original as in my change (previous change changed them).
The spaces in lines 33 and 35 are an indentation fix of something not present in the original repo but added in previous change.
The spaces on lines 22–27 and 39 are added due to this change but I think that might be preferred to get proper spacing for gradle files as only lines 22, 25, 26, 27 are without other changes.

Reply:
Okay, clear